### PR TITLE
UX SLDS updates to Schedule Payments page

### PR DESCRIPTION
**Pull request by seanpat09**
# Critical Changes

# Changes

# Issues Closed

# New Metadata

# Deleted Metadata

### CODE REVIEW

**Comment by seanpat09** [comment]
**lurch: attach W-029474

**Comment by LurchTheButler** [comment]
Tracking <a href="https://foundation.my.salesforce.com/a2x1E000007fPAbQAM" target="_blank">W-029474</a>

File: src/labels/CustomLabels.labels [hunk 1]
```diff
@@ -4516,6 +4524,22 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <shortDescription>pmtWizardInterval</shortDescription>
         <value>Interval</value>
     </labels>
+    <labels>
+        <fullName>pmtWizardIntervalNumber</fullName>
+        <categories>Payment Wizard</categories>
+        <language>en_US</language>
+        <protected>false</protected>
```
Comment (force2b) [inline src/labels/CustomLabels.labels]: @seanpat09 The new labels should have protected=true.

**Comment by seanpat09** [comment]
@force2b I've made the labels protected, let me know if there are any other changes needed!

**Comment by force2b** [comment]
Looking pretty good @seanpat09. Question though. Is it possible to align the payments picklist field with the interval fields so they're on the same line visually?
image

**Comment by seanpat09** [comment]
> Looking pretty good @seanpat09. Question though. Is it possible to align the payments picklist field with the interval fields so they're on the same line visually?

I brought this up to @mpusto in <a href="https://foundation.my.salesforce.com/a2x1E000007fPAbQAM" target="_blank">W-029474</a> that I was worried that I would spin my wheels on this too long and she was ok with the alignment as is. 

However, since we're at the end of the sprint I have a little time to dig into this a little more to see if I can find a SLDS friendly way of doing that without just using padding/margin. I worry if just throw some padding/margin in there then a change in SLDS could cause it to look even more unaligned.

**Comment by seanpat09** [comment]
@force2b I was able to get the alignment working, it looks much better now!

screen shot 2018-12-06 at 9 52 46 am

**Comment by force2b** [comment]
@seanpat09 What's the status on this PR? Is it going to @rponti-sforg for QA?

**Comment by seanpat09** [comment]
@force2b ah my mistake, I forgot to tag @rponti-sforg.

@rponti-sforg this is ready for QA!

**Comment by rponti-sforg** [comment]
QE testing is complete for this PR.

For further testing details, see: https://foundation.lightning.force.com/lightning/r/agf__ADM_Work__c/a2x1E000007fPAbQAM/view

Merging PR and deleting branch.

**Pull request merged**